script/prepare_artifacts.sh

```diff
@@ -14,11 +14,6 @@ for file in ./deployment/**/flyte_generated.yaml; do
     fi
 done
 
-# Normalize VERSION if it's a beta (e.g., v1.16.0b0 → v1.16.0-beta.0)
```

Comment:
@pmahindrakar-oss I removed this, since we should always release with tag v1.16.0-b0.